Readme.md

```diff
@@ -20,21 +20,13 @@ Manual install:
 2. [Download Outer Wilds mods](https://outerwildsmods.com/mods) and put them in the Mods folder, each mod in a separate folder.
 3. Start the game with OWML.Launcher.exe.
 
-## Sample mod
-
-One mod is included as an example. It's disabled by default, enable in manifest.json.
-
-|Sample mod|Description|
-|----------|-----------|
-|OWML.EnableDebugMode|Enables the built-in debug mode in the game. Highlights: cycle through debug UIs with F1, warp to planets with the number keys, and explode the sun with the End key.|
-
 ## For modders
 
-Refer to the sample mods for examples.
+Refer to the sample mods in the source code for examples. These mods are not included in releases.
 
 ### Get started
 
-1. Create a class library project targeting .Net Framework 3.5.
+1. Create a C# class library project targeting .Net Framework 3.5.
```

Comment:
.Net Core now, right?

Reply:
nope. the project files are based on .net core but it's still .net framework. Can't change that because of OW/Unity.